.drone.yml

```diff
@@ -63,6 +63,8 @@ steps:
   volumes:
   - name: cache
     path: /go
+  - name: dockersock
```

Comment:
this (the droneio pipeline) is not working, I am not sure containers in the build are given privileges to access the docker daemon, are they?

Reply:
No, I don't believe so. You get access to the daemon in GitHub actions, though (#1460). Would it make sense to go that route?

Reply:
Let me do a little investigating here 🤔 will get back to you shortly

Reply:
@arschles I have added the workflows directory, based off of your gh-actions branch, here is the latest action running the tests (with testcontainers) https://github.com/ClaytonNorthey92/athens/actions/runs/42204895